deep_quoridor/src/train_dqn.py

```diff
@@ -59,11 +62,14 @@ def train_dqn(
             observation, reward, termination, truncation, _ = game.last()
 
             # If the game is over, break the loop
+            # BUG: Missing recording a negative reward for the DQN agent when the opponent wins
             if termination or truncation:
+                if False:
+                    print(f"Winner: {game.get_opponent(agent_name)}. Episode reward: {episode_reward}")
```

Comment:
We should use a logging framework so we can keep this things as `log.debug` or something and we can turn on and off from the command line